app/forms/trainee_id_form.rb

```diff
@@ -3,11 +3,7 @@
 class TraineeIdForm < TraineeForm
   attr_accessor :trainee_id
 
-  validates :trainee_id, presence: true,
-                         length: {
-                           maximum: 100,
-                           message: I18n.t("activemodel.errors.models.trainee_id_form.attributes.trainee_id.max_char_exceeded"),
-                         }
+  validates :trainee_id, presence: true
```

Comment:
You'll need to bring back the `length` argument. `trainee_id` is not a foreign key. It's a string the user will enter. I also removed this `length` argument recently by accident. Wondering if we should rename `trainee_id` to something better - it's kinda going against the Rails convention with regards to foreign key names. cc: @AbigailMcP @gpeng